jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/logging/ConsoleLoggerBuilder.java

```diff
@@ -47,9 +47,9 @@
   public static ConsoleLoggerBuilder rich(
       SingleThreadedExecutor singleThreadedExecutor, boolean enableTwoCursorUpJump) {
     return new ConsoleLoggerBuilder(
-        messageConsumers ->
+        levelMessageConsumerMap ->
```

Comment:
This sounds a bit weird. Just `messageConsumerMap`?

Reply:
yea. This sounds better.